src/pages/DashboardPage.jsx

```diff
@@ -1,5 +1,48 @@
-function Dashboard() {
-  return <h2>Dashboard</h2>;
-}
+import Table from "../components/Table";
+
+const rows = [
+  { description: "ddd", amount: 111, expenseGroup: "123", id: "1" },
+  { description: "p", amount: 123, expenseGroup: "1", id: "12" },
+  { description: "djurdja", amount: 100, expenseGroup: "13", id: "22" },
+  { description: "abc", amount: 100, expenseGroup: "13", id: "24" },
+  { description: "def", amount: 200, expenseGroup: "8", id: "23" },
+];
+const columns = [
```

Comment:
We could extract somewhere table configuration so that our component will be more readable and just import it from another place. You can create maybe a common folder where you can have different reusable things such as table config for example.